app/assets/stylesheets/admin/attachment.scss

```diff
@@ -0,0 +1,179 @@
+// Place all the styles related to the admin/attachment controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+@import "font-awesome";
+
+$border-color: #003087;
```

Comment:
Are these colors used in other pages?
If so, then you have a variables file where all common colors are written to variables.

Reply:
fixed